index.js

```diff
@@ -90,18 +90,47 @@ const exec = (cmd, args = []) =>
  * @returns {Promise<void>}
  */
 
-const commitFile = async () => {
+const commitFile = async (emptyCommit = false) => {
   await exec("git", ["config", "--global", "user.email", COMMIT_EMAIL]);
   await exec("git", ["config", "--global", "user.name", COMMIT_NAME]);
-  await exec("git", ["add", TARGET_FILE]);
-  await exec("git", ["commit", "-m", COMMIT_MSG]);
+  if (emptyCommit) {
+    await exec("git", ["commit", "--allow-empty", "-m", EMPTY_COMMIT_MSG]);
+  } else {
+    await exec("git", ["add", TARGET_FILE]);
+    await exec("git", ["commit", "-m", COMMIT_MSG]);
+  }
   await exec("git", ["push"]);
 };
 
+/**
+ * Make empty commit
+ * @returns {Promise<void>}
+ * */
+const emptyCommit = async () => {
+  const { outputData } = await exec(
+    "git",
+    ["--no-pager", "log", "-1", "--format=%ct"],
+    { encoding: "utf8", stdio: ["pipe", "pipe", "pipe"] },
+  );
+
+  const commitDate = new Date(parseInt(outputData, 10) * 1000);
```

Comment:
The exec function does not have a return value:

Line 81 needs to be extended to return the stdout.

```
return resolve(stdout);
```

Unfortunately, this would cause an output to be rendered for every other `await exec(...)` as well. Maybe that is even a good thing 🤔

Reply:
This will still fail now, because you get an object with two fields and try to parse it into an integer

Reply:
May I make some changes to the PR myself?

Reply:
Sure